sql/schemas/bookbrainz.sql

```diff
@@ -810,21 +810,24 @@ ALTER TABLE bookbrainz.user_collection_collaborator ADD FOREIGN KEY (collaborato
 
 CREATE VIEW bookbrainz.author AS
 	SELECT
-		e.bbid, cd.id AS data_id, cr.id AS revision_id, (cr.id = c.master_revision_id) AS master, cd.annotation_id, cd.disambiguation_id,
-		als.default_alias_id, cd.begin_year, cd.begin_month, cd.begin_day, cd.begin_area_id,
+		e.bbid, cd.id AS data_id, cr.id AS revision_id, (cr.id = c.master_revision_id) AS master, cd.annotation_id, dis.comment,
+		als.default_alias_id, al."name", al.sort_name, cd.begin_year, cd.begin_month, cd.begin_day, cd.begin_area_id,
 		cd.end_year, cd.end_month, cd.end_day, cd.end_area_id, cd.ended, cd.area_id,
-		cd.gender_id, cd.type_id, cd.alias_set_id, cd.identifier_set_id, cd.relationship_set_id, e.type
+		cd.gender_id, cd.type_id, atype.label as author_type, cd.alias_set_id, cd.identifier_set_id, cd.relationship_set_id, e.type
 	FROM bookbrainz.author_revision cr
 	LEFT JOIN bookbrainz.entity e ON e.bbid = cr.bbid
 	LEFT JOIN bookbrainz.author_header c ON c.bbid = e.bbid
 	LEFT JOIN bookbrainz.author_data cd ON cr.data_id = cd.id
 	LEFT JOIN bookbrainz.alias_set als ON cd.alias_set_id = als.id
+	LEFT JOIN bookbrainz.alias al ON al.id = als.default_alias_id
+	LEFT JOIN bookbrainz.disambiguation dis ON dis.id = cd.disambiguation_id
+	LEFT JOIN bookbrainz.author_type atype ON atype.id = cd.type_id
 	WHERE e.type = 'Author';
```

Comment:
this review returns one row for each revision of an entity. I think that we want to treat the view as a snapshot of the current state of the database, so it probably makes sense to add `AND master=true` to all of the where statements here (and perhaps that means it makes sense to remove `master` from the returned column list?

Reply:
I looked into this more, and there are a few places in the website and ORM where we use these views to fetch older revision.
It can probably be done without too much problems, but will require a lot of carefulness and time to test everything (lest we break the whole website somehow).
Leaving this for another time.